packages/main/src/StepInput.js

```diff
@@ -0,0 +1,645 @@
+		 *
+		 * @type {Float}
+		 * @defaultvalue 1
+		 * @public
+		 */
+		step: {
+			type: Float,
+			defaultValue: 1,
+		},
+
+		/**
+		 * Defines the value state of the <code>ui5-step-input</code>.
+		 * <br><br>
+		 * Available options are:
+		 * <ul>
+		 * <li><code>None</code></li>
+		 * <li><code>Error</code></li>
+		 * <li><code>Warning</code></li>
+		 * <li><code>Success</code></li>
+		 * <li><code>Information</code></li>
+		 * </ul>
+		 *
+		 * @type {ValueState}
+		 * @defaultvalue "None"
+		 * @public
+		 */
+		valueState: {
+			type: ValueState,
+			defaultValue: ValueState.None,
+		},
+
+		/**
+		 * Defines whether the <code>ui5-step-input</code> is required.
+		 *
+		 * @type {Boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		required: {
+			type: Boolean,
+		},
+
+		/**
+		 * Determines whether the <code>ui5-step-input</code> is displayed as disabled.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+
+		/**
+		 * Determines whether the <code>ui5-step-input</code> is displayed as read-only.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		readonly: {
+			type: Boolean,
+		},
+
+		/**
+		 * Defines a short hint, intended to aid the user with data entry when the
+		 * <code>ui5-step-input</code> has no value.
+		 *
+		 * <br><br>
+		 * <b>Note:</b> When no placeholder is set, the format pattern is displayed as a placeholder.
+		 * Passing an empty string as the value of this property will make the <code>ui5-step-input</code> appear empty - without placeholder or format pattern.
+		 *
+		 * @type {string}
+		 * @defaultvalue undefined
+		 * @public
+		 */
+		placeholder: {
+			type: String,
+			defaultValue: undefined,
+		},
+
+		/**
+		 * Determines the name with which the <code>ui5-step-input</code> will be submitted in an HTML form.
+		 *
+		 * <br><br>
+		 * <b>Important:</b> For the <code>name</code> property to have effect, you must add the following import to your project:
+		 * <code>import "@ui5/webcomponents/dist/features/InputElementsFormSupport.js";</code>
+		 *
+		 * <br><br>
+		 * <b>Note:</b> When set, a native <code>input</code> HTML element
+		 * will be created inside the <code>ui5-step-input</code> so that it can be submitted as
+		 * part of an HTML form. Do not use this property unless you need to submit a form.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		name: {
+			type: String,
+		},
+
+		/**
+		 * Determines the number of digits after the decimal point of the <code>ui5-step-input</code>.
+		 *
+		 * @type {Integer}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		valuePrecision: {
+			type: Integer,
+			defaultValue: 0,
+		},
+
+		/**
+		 * Defines the aria-label attribute for the <code>ui5-step-input</code>.
+		 *
+		 * @type {String}
+		 * @private
+		 * @defaultvalue ""
+		 */
+		ariaLabel: {
```

Comment:
Both ariaLabel and ariaLabelledby doesn't make anything, you have to forward them to the internal Input component, use the DatePicker as reference particularly the method
`get accInfo() {`
Otherwise nothing would be read.

Reply:
Done